p2p_wallet/Scenes/Main/WithdrawActions/WithdrawActionsView.swift

```diff
@@ -0,0 +1,77 @@
+import Combine
+import KeyAppUI
+import SwiftUI
+
+struct WithdrawActionsView: View {
+    @ObservedObject var viewModel: WithdrawActionsViewModel
+
+    var body: some View {
+        VStack(spacing: 8) {
+            RoundedRectangle(cornerRadius: 2, style: .circular)
+                .fill(Color(Asset.Colors.rain.color))
+                .frame(width: 31, height: 4)
+                .padding(.top, 6)
+            Text(L10n.withdraw)
+                .fontWeight(.semibold)
+                .apply(style: .title3)
+                .padding(.top, 22)
+                .padding(.bottom, 20)
+            ForEach(viewModel.actions) { item in
+                Button {
+                    viewModel.didTapItem(item: item)
```

Comment:
what about subject? we discussed recently to use it instead of methods

Reply:
I don't want to put it here, since I need a whole item to be passed